### DISCUSSION TITLE
Unable to connect with our database

### DISCUSSION THREAD

**Discussion by justincaze** [neuefische/questions#814] [Web] [answered]
We're unable to connect to our database and we really don't know why.

This is the Error:

Unhandled Runtime Error
TypeError: can't access property "Activity", mongoose__WEBPACK_IMPORTED_MODULE_0___default().models is undefined

And there is our code:

https://github.com/alex96dev/Easy-Meetup-with-Friends/tree/activity_list

**Comment by NikRoe** [comment, marked as answer]
We solved this issue directly in Zoom.

In `/pages/index.js` we needed to change
```js
const { activities, isLoading } = useSWR("/api/activities");
```

to be 
```js
const { data: activities, isLoading } = useSWR("/api/activities");
```

for the code to execute properly.
